### ISSUE TITLE
Training Details on RealSR Dataset

### ISSUE THREAD

**Issue by zhaob10** [swz30/MIRNet#26]
Hi! The performance of your method on the RealSR datasets is really attractive. I am trying to train the model by following the instructions in the paper, but have some difficulties reproducing the results. The RealSR dataset totally contains 505 image pairs for training and 30 image pairs for validation on each scale. In the training stage, I choose to crop 320 patches with size 128×128 from one image. Therefore, the number of patches in the training set is 505×320. It seems that the training dataset is very large such that it takes very long (about 4.5 hours) to process one epoch with batch size 8 on a GPU with 24GB memory.  Could I know the details of the training epoch, patch size and the total number of patches in the RealSR training dataset? Besides, do you train the model on the whole dataset and validate the performance on the validation set at each scale? Looking forward to your reply! Thank you!

**Comment by zhaob10** [comment]
Dear author,
Thank you for your great work and the results look quite good. I also try to reproduce the results of the real world super resolution experiments. I saw the CharbonnierLoss is applied in the denoising training code. Does this loss function is also used for training on the RealSR dataset?
